packages/calcite-components/src/components/input/resources.ts

```diff
@@ -11,7 +11,6 @@ export const CSS = {
   wrapper: "element-wrapper",
   inputWrapper: "wrapper",
   actionWrapper: "action-wrapper",
-  resizeIconWrapper: "resize-icon-wrapper",
```

Comment:
should this be part of this PR?

Reply:
I removed the corresponding element in the component.

For context - this was a custom "grab handle" the now-deprecated `type="textarea"` used in place of the browser-rendered ones. We use the latter in the newer Text Area component so this brings this into alignment and saves some workarounds we'd need to do. 

cc @jcfranco - I can remove this separately if needed.

Reply:
Expected Chromatic diff: https://www.chromatic.com/test?appId=6266d45509d7eb004aa254fb&id=673e5eb73fed785bcc02cf6e

Reply:
If we don't need to handle this separately we can accept that diff. 👍

Reply:
I think we'd want to include it here because without this change the previous resize handle affordance "breaks" with any corner radius overrides.

Reply:
Let's submit the handle change separately to keep this PR focused on tokens. We can handle the other PR as a feat, using Adam's alignment description. Is there an existing issue we can use, or do we need a new one?

Reply:
https://github.com/Esri/calcite-design-system/issues/10864